utilities/test_suite/HOST/Tensor_host.cpp

```diff
@@ -511,6 +511,44 @@ int main(int argc, char **argv)
 
                 break;
             }
+            case 34:
+            {
+                testCaseName = "lut";
+
+                Rpp32f lutBuffer[65536] = {0};
+                Rpp8u *lut8u = reinterpret_cast<Rpp8u *>(lutBuffer);
+                Rpp16f *lut16f = reinterpret_cast<Rpp16f *>(lutBuffer);
+                Rpp32f *lut32f = reinterpret_cast<Rpp32f *>(lutBuffer);
+                Rpp8s *lut8s = reinterpret_cast<Rpp8s *>(lutBuffer);
+                if (inputBitDepth == 0)
+                    for (j = 0; j < 256; j++)
+                        lut8u[j] = (Rpp8u)(255 - j);
+                else if (inputBitDepth == 3)
+                    for (j = 0; j < 256; j++)
+                        lut16f[j] = (Rpp16f)((255 - j) * ONE_OVER_255);
```

Comment:
probably you should do division here so we don't loose precision

Reply:
This is actually just a sample case in the test suite for users to see some example. The user could always modify to give any LUT of their choice, its not part of the functionality